CODEOWNERS

```diff
@@ -0,0 +1,19 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+* @ChShersh
+
+# Order is important. The last matching pattern has the most precedence.
+# So if a pull request only touches hlint.yaml file, only these owners
```

Comment:
`touches just the hlint.yaml file` if it touches only that file, not other files, and that matters
`just touches the hlint.yaml file` if it could do something else to the file, except touching, and that matters 
`touches hlint.yaml file, these owners` if it doesn't matter whether something else happens in the PR